packages/nakama-js/utils.ts

```diff
@@ -1,5 +1,57 @@
 import {encode, decode} from "js-base64"
 
+export function buildFullUrl(basePath: string, fragment: string, queryParams: Map<string, any>)
```

Comment:
I think we should inline these functions as part of the output of the codegen tool so that the tool's output is always standalone. This makes it easier to fulfil one of our later goals to consolidate our codegen utils under a single repo as an official CLI.